moonbeam-types-bundle/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "moonbeam-types-bundle",
-  "version": "2.0.1",
+  "version": "2.0.2",
   "description": "Bundled types to instantiate the Polkadot JS api with a Moonbeam network",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "prepublish": "tsc",
   "scripts": {
     "compare": "rm -fr dist && tsc && npx ts-node check.ts",
     "build": "rm -fr dist && node_modules/.bin/tsc",
-    "publish-package": "tsc && npm publish",
+    "publish-package": "npm install && npm publish",
```

Comment:
I'm not a big fan of that

Reply:
If you insist on installing it, it should at least build it after to see the changes :/

Reply:
ok thats a leftover of when npm install used to build